backend/chain.go

```diff
@@ -368,6 +368,84 @@ func (g *gnoZenaoChain) UserAddress(userID string) string {
 	return gnolang.DerivePkgAddr(g.userRealmPkgPath(userID)).String()
 }
 
+// CreatePoll implements ZenaoChain
+func (g *gnoZenaoChain) CreatePoll(userID string, req *zenaov1.CreatePollRequest) error {
+	userRealmPkgPath := g.userRealmPkgPath(userID)
+	eventPkgPath := g.eventRealmPkgPath(req.EventId)
+	feedID := gnolang.DerivePkgAddr(eventPkgPath).String() + ":main"
+	options := ""
+	for _, option := range req.Options {
+		options += fmt.Sprintf(`%q, `, option)
+	}
+	kind := ""
+	if req.MultipleChoices {
+		kind = "pollsv1.POLL_KIND_MULTIPLE_CHOICE"
```

Comment:
this seem weird, should be `POLL_KIND_MULTIPLE_CHOICE`, no?

Reply:
[19c62e6](https://github.com/samouraiworld/zenao/pull/282/commits/19c62e69954f04a9ff597510c8552944c431f3b2)